roles/kubernetes/files/certs/ca.crt

```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
yep, all the certs and keys can be overridden by setting `k8s_local_cert_dir` variable. The bundled certs will be helpful for bringing up test and demo environments